### PR TITLE
P1673R10 revisions

### PR DESCRIPTION
**Pull request by mhoemmen**
@crtrott and I went through part of the P1673 wording today.  We fixed some things and marked others for later revision.  We have more work to do yet before this should be submitted as P1673R10.

* Revise `scaled` and `conjugated` wording.
* Make all matrix view functions `constexpr`.

### CODE REVIEW

**Comment by crtrott** [comment]
Ah thanks for issuing the pr, i think its good to do this piecemeal

**Comment by mhoemmen** [comment]
@crtrott Regarding paragraph numbers: I think I'll just convert to Pandoc Markdown, like P0009, but it seems wise to do that in a separate PR once we've finished content revisions.

**Comment by crtrott** [comment]
Did you try whether you can just add numbers in bs?

**Review by crtrott — COMMENTED** [review]
Do we want to merge this first and then fix more? For example we need to replace pointer with data_handle

File: D1673/P1673.bs [hunk 1]
```diff
@@ -5878,106 +5711,112 @@ It also makes fewer requirements of the reference type.
...
 
-  accessor_scaled(ScalingFactor s, Accessor a);
+  constexpr accessor_scaled(const ScalingFactor& s, const Accessor& a);
 
-  template<class OtherElementType>
-  accessor_scaled(ScalingFactor s, default_accessor<OtherElementType> a);
+  constexpr reference access(pointer p, size_t i) const noexcept;
 
-  reference access(pointer p, size_t i) const noexcept;
+  constexpr offset_policy::pointer offset(pointer p, size_t i) const noexcept;
```
[the remark this answers is not in the document]
Comment (mhoemmen) [reply D1673/P1673.bs:5768]: Thanks Christian!  I've fixed this in PR https://github.com/ORNL/cpp-proposals-pub/pull/301 .

**Pull request merged**